tests/Test.py

```diff
@@ -0,0 +1,24 @@
+import os
```

Comment:
Can we name this file something other than test.py? There is also a regression test package named 'test' - https://docs.python.org/3/library/test.html#module-test.support

Reply:
`TestBase`?